### DISCUSSION TITLE
re hue  tuya strip lights

### DISCUSSION THREAD

**Discussion by oliverchapple** [hass-emulated-hue/core#102] [General]
hi 

do you thin k this is a bug.  when I add the strip lights by tuya it says saving. clould take a while please keep the app open unit saveing is complete hue. don't know. if you seen that and how to fix it

thanks Oliver

**Comment by alexyao2015** [comment]
Huh? Explain?

**Comment by oliverchapple** [comment]
the Alexa was picking up the devices 

but now it not    after home assistant update 

it worked before

**Reply by alexyao2015** [reply]
We don't support cloud devices. If you need that, look at home assistant's own alexa integration. Don't try to use the hue emulator with any cloud services
